### PR TITLE
quickstart: How to use a different KBC with enclave-cc

### PR DESCRIPTION
**Pull request by fidencio**
Let's add a small piece of documentation about what the users should do in case they want to try enclave-cc with a different KBC.

Signed-off-by: Fabiano Fidêncio <fabiano.fidencio@intel.com>

### CODE REVIEW

**Review by mythi — APPROVED** [review]

File: quickstart.md [hunk 1]
```diff
@@ -146,6 +146,24 @@ NAME            HANDLER         AGE
 enclave-cc      enclave-cc      9m55s
 ```
 
+### Configuring enclave-cc custom resource to use a different KBC
+
+**Note** This must be done **before** 
+
+Enclave CC supports Verdictd and in order to use it, users will have to
+*properly configure a decrypt_config.conf, in order to set the `KBC` (`sample_kbc`
```
Comment (dcmiddle) [inline quickstart.md:154]: Extraneous *
```suggestion
properly configure a decrypt_config.conf, in order to set the `KBC` (`sample_kbc`
```

File: quickstart.md [hunk 2]
```diff
@@ -146,6 +146,24 @@ NAME            HANDLER         AGE
 enclave-cc      enclave-cc      9m55s
 ```
 
+### Configuring enclave-cc custom resource to use a different KBC
+
+**Note** This must be done **before**
```
Comment (dcmiddle) [inline quickstart.md:151]: before ... what? :)
```suggestion
**Note** The KBC configuration changes to the enclave-cc custom resource yaml 
must be made **before** deploying it. 
```

**Review by dcmiddle — CHANGES_REQUESTED** [review]

**Comment by fidencio** [comment]
Updated, thanks @dcmiddle!

**Review by dcmiddle — APPROVED** [review]

**Review by fitzthum — APPROVED** [review]
This is good but I think it's in the wrong section. The doc is getting pretty messy but I think we probably want this in `Creating a sample Coco workload using enclave-cc` or farther down in one of the sections about using encrypted images.

**Comment by fidencio** [comment]
> This is good but I think it's in the wrong section. The doc is getting pretty messy but I think we probably want this in `Creating a sample Coco workload using enclave-cc` or farther down in one of the sections about using encrypted images.

I went back and forth on this one, @fitzthum.  The reason I decided to put this in this section is because this must be changed when deploying the custom resource.

I imagine myself doing all the steps, then noticing when deploying a sample workload that the custom resource has to change, that wouldn't make me happy. :-).

I can move it down if you prefer, just let me know.

**Comment by dcmiddle** [comment]
I'd suggest leaving it here for this release but then in 0.4 we redo this whole document. 
Basically make the existing document a general user guide and clip down to just a few lines for an operator quickstart (and maybe figure out a separate quickstart for deploying/managing a container that assumes a running coco infrastructure).

**Comment by fitzthum** [comment]
Well maybe we could put a note about it in the first section that links to the later section?

**Comment by dcmiddle** [comment]
> Well maybe we could put a note about it in the first section that links to the later section?

I'm fine with that too.

**Comment by fidencio** [comment]
> Well maybe we could put a note about it in the first section that links to the later section?

I still think it'll be confusing as that part is not related to the workload itself.

**Comment by fitzthum** [comment]
I think the doc is trying to be a bit incremental, starting with general and simple stuff like running unencrypted containers or sample containers (mirroring the 4 bullets at the very top). It seems like we would only need to switch to a different KBC when we start running encrypted or signed workloads. It's a good idea to alert people in the beginning since they might have to do it before deploying the CR, but I think we want to limit the amount of stuff that someone has to read before they get to the part where they start the first container. So I would just do a link to a later section, but honestly I don't care that much.

We probably should figure out how to make the quickstart guide a bit quicker for the next release, although I'm not exactly sure how.

**Comment by fidencio** [comment]
> I think the doc is trying to be a bit incremental, starting with general and simple stuff like running unencrypted containers or sample containers (mirroring the 4 bullets at the very top). It seems like we would only need to switch to a different KBC when we start running encrypted or signed workloads. It's a good idea to alert people in the beginning since they might have to do it before deploying the CR, but I think we want to limit the amount of stuff that someone has to read before they get to the part where they start the first container. So I would just do a link to a later section, but honestly I don't care that much.

I'll proceed with this as is, as you don't care that much, with the promise that we will revisit it as soon as this release is over.

> 
> We probably should figure out how to make the quickstart guide a bit quicker for the next release, although I'm not exactly sure how.

In the last Kata Containers meeting, we had a discussion about using a `k3s` style of documentation, which I think would fit very well for our case.  However, it'd be better outside of GitHub.

This is an example: https://docs.k3s.io/

**Pull request merged**